projects/storefrontlib/src/cms-components/navigation/search-box/search-box.component.ts

```diff
@@ -46,7 +51,8 @@ export class SearchBoxComponent {
    * for example when we click inside the search result section.
    */
   private ignoreCloseEvent = false;
-
+  public chosenWord = '';
+  public updateChosenWordFromUrl: Subscription;
```

Comment:
let's keep the variable naming consistent as it is throughout the application

change to `subscription`

Reply:
just putting the link to the above comment https://github.com/SAP/spartacus/pull/8298#discussion_r568719587 as you might miss it